tools/changelog-generator/README.md

```diff
@@ -27,14 +27,17 @@ To generate a `CHANGELOG.md` file for all released Kyma versions, follow these s
 1. Create a `CHANGELOG.md` file under the specified absolute path to the repository. Use this command:
 
 ```bash
-docker run --rm -v {absolutePathToRepository}:/repository -w /repository -e LATEST_VERSION={applicationVersion} -e GITHUB_AUTH={githubToken} -e SSH_FILE={sshFilePath} changelog-generator sh /app/generate-full-changelog.sh --configure-git
+docker run --rm -v {absolutePathToRepository}:/repository -w /repository -e LATEST_VERSION={applicationVersion} -e GITHUB_AUTH={githubToken} -e SSH_FILE={sshFilePath} -e CONFIG_FILE={configFilePath} -e SKIP_REMOVING_LATEST=true changelog-generator sh /app/generate-full-changelog.sh --configure-git
 ```
 
 Replace values in curly braces with proper details, where:
 - `{absolutePathToRepository}` is the absolute path to the repository.
 - `{applicationVersion}` is the currently released application version.
 - `{githubToken}` is the GitHub API token with the read-only access to the repository.
 - `{sshFilePath}` is the path to the SSH file used for Git to authenticate with the repository.
+- `{configFilePath}` is the path to file used by `lerna-changelog` to repository configuration
+
+Add `SKIP_REMOVING_LATEST=true` variable if you want skip removing 'latest' tag functionality.
```

Comment:
Please describe configuration for `SKIP_REMOVING_LATEST like in above paragraph.

So in the example above it should be something like 
`-e SKIP_REMOVING_LATEST={skipRemovingLatest}` and then below you describe the role of the variable